src/components/Router/StepsRouter.tsx

```diff
@@ -59,13 +60,22 @@ class StepsRouter extends Component<StepsRouterProps> {
       ? cobrand
       : globalUserOptions.enterpriseFeatures?.cobrand && cobrand
 
+    const logoCobrandLogic = mobileFlow
+      ? logoCobrand
+      : globalUserOptions.enterpriseFeatures?.logoCobrand && logoCobrand
+
+    const logoCobrandStyle = logoCobrandLogic
+      ? { backgroundImage: 'url(' + logoCobrandLogic.src + ')' }
+      : null
+
     return (
       //TODO: Wrap CurrentComponent in themeWrap HOC
       <div
         className={classNames(theme.step, {
           [theme.fullScreenStep]: isFullScreen,
           [theme.noLogo]: hideLogoLogic,
           [theme.cobrandLogo]: cobrandLogic,
+          [theme.logoCobrandLogo]: logoCobrand,
```

Comment:
I think this needs a better/clearer name to distinguish between the "text only" cobrand logo vs this "full image replacement" cobrand logo. If needed, it's OK to rename `cobrandLogo` also.

Reply:
Updated cobrandLogo to be textCobrandLogo, let me know if that still isn't clear